src/status_im2/navigation/view.cljs

```diff
@@ -117,6 +117,18 @@
       (when config/keycard-test-menu-enabled?
         [keycard.test-menu/test-menu])])))
 
+(def sheet-comp2
```

Comment:
we don't need to support old component anymore, so new one should be used in `sheet-comp` no need in 2